dist/types.d.ts

```diff
@@ -0,0 +1,118 @@
+export interface RenderedQuery {
```

Comment:
we should be able to stop checking in `dist` changes for now. the github releases will contain the `dist` folder that is created as part of `yarn build`

Reply:
+1

Reply:
+1